deploy/templates/bicep/UploadTemplates.bicep

```diff
@@ -0,0 +1,151 @@
+@minLength(3)
+@maxLength(16)
+@description('Basename that is used to name provisioned resources. Should be alphanumeric, at least 3 characters and less than 16 characters.')
+param baseName string
+
+@description('Location where the resources are deployed. For a list of Azure regions where Azure Health Data Services are available, see [Products available by regions](https://azure.microsoft.com/explore/global-infrastructure/products-by-region/?products=health-data-services)')
+@allowed([
+  'australiaeast'
```

Comment:
Suggestion: Is there a way to share a common location list between the bicep templates?  It would simplify adding new locations.

Reply:
Thanks for bringing this up, I think this is a great idea. I tried creating a separate JSON file to store a list of the deployment options and loading it into the bicep templates using loadJsonContent() in the allowed decorator for the location parameter. However, I got an error saying that the allowed values must be available at compile time. There is also a way to pass in allowed values by creating a user-defined type, but that would require enabling a preview feature. I will look into other possible options.